pkg/kubectl/cmd/rollingupdate/rollingupdate.go

```diff
@@ -356,6 +356,9 @@ func (o *RollingUpdateOptions) Run() error {
 
 	if o.Rollback {
 		newName := o.FindNewName(oldRc)
+		if len(newName) == 0 {
+			return fmt.Errorf("Could not rollback a succeed roling-update or a non-rolling-update RC")
```

Comment:
Is it correct to change "roling-update" to "rolling-update"?

Reply:
Thankyou , i have changed it